content/docs/user-guide/visualization/environments/terrain/terrain-developer-guide/future-work.md

```diff
@@ -0,0 +1,55 @@
+---
+linktitle: Future Work
+title: Future Work
+description: Possible future improvements to the terrain system.
+weight: 500
+toc: true
+---
+
+There are a number of improvements that could potentially be made to the terrain system for anyone interested in taking the system further.
```

Comment:
```suggestion
There are a number of improvements that can potentially be made to the terrain system for anyone interested in taking the system further. You can reach out to the O3DE community to talk through any terrain enhancements or ideas on the #sig-content channel on [Discord ](https://discord.com/invite/o3de). We are happy to help!
```

Reply:
How's that sound?

Reply:
Hm, I was hoping my editing on your suggestion would show so you can see it. (I noticed that works in GH issues.) So apologies for lack of transparency there. I made two small changes, changed "talk through" to "talk about", and removed a space after "Discord".